Microsoft.Toolkit.Uwp.UI.Controls.Primitives/Microsoft.Toolkit.Uwp.UI.Controls.Primitives.csproj

```diff
@@ -0,0 +1,46 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+
+	<PropertyGroup>
+		<TargetFrameworks>uap10.0.17763</TargetFrameworks>
+		<Title>Windows Community Toolkit Controls</Title>
+		<Description>
+			This library provides XAML templated controls. It is part of the Windows Community Toolkit.
+
+			Controls:
+			- AdaptiveGridView: Presents items in an evenly-spaced set of columns to fill the total available space.
+			- DockPanel: Define areas where you can arrange child elements either horizontally or vertically, relative to each other.
+			- StaggeredPanel: Layout of items in a column approach where an item will be added to whichever column has used the least amount of space.
+			- UniformGrid: Presents items in an evenly-spaced set of rows or columns to fill the total available display space.
```

Comment:
Add:
SwitchPresenter: A presenter which can act as a switch statement for layout providing alternate layouts based on a value.

Reply:
Done ac71f64